queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon';
+select * from animals where date_of_birth BETWEEN '1-1-2016' AND '1-1-2019';
```

Comment:
- [ ] Kindly be noticed that when you wrote the date `1-1-2019` you didn't include `2019` year it should be written like this `select * from animals where date_of_birth BETWEEN '1-1-2016' AND '31-12-2019';`